setup/methylationqc/data/meth_hg38-chr20_Lambda_pUC19.interval_list

```diff
@@ -0,0 +1,461 @@
+@HD	VN:1.6	SO:coordinate
+@SQ	SN:chr1	LN:248956422	M5:2648ae1bacce4ec4b6cf337dcae37816	UR:file:///data/hg38_Lambda_pUC19.fa
```

Comment:
@GatKrieger, maybe we should remove all @SQ (or keep just the ones relevant for the intervals in this list?)